sql/ddl_internal.sql

```diff
@@ -1,3 +1,16 @@
+-- Q:: this is now only used in tests to do sth like show_chunks:
```

Comment:
Yes this should be removed from here. Please add it in the test script if necessary